### ISSUE TITLE
[Bug] cannot load such file -- onfido/null_logger

### ISSUE THREAD

**Issue by ctrlaltdylan** [onfido/onfido-ruby#24]
Issues when booting up Rails v6.1 on the onfido-ruby v2.0.0 client installed:

```
/usr/local/bundle/gems/bootsnap-1.7.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require': cannot load such file -- onfido/null_logger (LoadError)
```

It looks like this was removed on master but I don't believe there has been a subsequent tag for `v2.0.1` to include this fix: https://github.com/onfido/onfido-ruby/commit/aa67f943b4adc7e170c6514a200ebc25185560e8

**Comment by superdevx64** [comment]
Hi @ctrlaltdylan apologies for this! Thank you very much for raising it; it's really appreciated.

We've released `v2.0.1`: please let us know here if you have any issues with the new release and please accept our apologies again.

**Comment by dvacca** [comment]
Hello @ctrlaltdylan,
We're going to close this issue as we didn't get any reply so far but we suppose released fixed is working fine.

Please reopen the issue if you're still experiencing that issue. Thanks!
